addons/hr_work_entry/models/hr_work_entry.py

```diff
@@ -112,6 +112,7 @@ def _mark_conflicting_work_entries(self, start, stop):
                     AND tsrange(b1.date_start, b1.date_stop, '()') && tsrange(b2.date_start, b2.date_stop, '()')
                     AND b1.id <> b2.id
                     AND b1.employee_id = b2.employee_id
+                    AND b1.company_id <> b2.company_id
```

Comment:
Hi @yosa-odoo 
I started an enterprise build on your branch to be able to test but I am pretty sure this completely breaks the conflict mechanism as work entries will pretty much never be marked as conflict.

I understand the issue and I propose this solution instead (for the first select line 102~): 
```python
AND b1.date_stop >= %s
AND b1.company_id in %s
AND active = TRUE
```
I added the `b1.company_id in %s` line so that we only check conflicts for the current active companies, of course you will need to give the company ids as arguments to the query 🙂 

What do you think?

cc @tivisse

Reply:
Hello @Williambraecky,

Thanks for the review. 
You're right, it totally breaks the check conflict.

I'm gonna investigate/test your solution.

Reply:
@Williambraecky well spotted ! :)

Thanks for your work @yosa-odoo